src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -16913,5 +16982,146 @@ struct S
                 //         s.P.ToString();
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "s.P").WithLocation(8, 9));
         }
+
+        [Fact]
+        public void UnassignedClassAutoProperty()
+        {
+            var source =
+@"class C
+{
+    object? P { get; }
+    C(out object o)
```

Comment:
>C(out object o) [](start = 4, length = 15)

It would be good to test similar scenario in a regular method.

Reply:
Added.